ansible_testing/modules.py

```diff
@@ -181,7 +181,11 @@ def _just_docs(self):
             return False
 
     def _is_new_module(self):
-        return not module_loader.has_plugin(self.name)
+        if self.name.startswith("_"):
```

Comment:
Thanks for the quick review :)

Do you literally mean:
`if self.name.startswith("_") and not os.path.islink(self.name):`?

I don't see any symlinks in ansible-modules-core (or extras), do you know what version they were in so I can test this?